contracts/MemeBase.sol

```diff
@@ -83,6 +97,10 @@ contract MemeBase is MemeFactory {
         address _l2TokenRelayer,
         address _balancerVault,
         bytes32 _balancerPoolId,
+        address _redemptionMemeBase,
+        uint256 _redemptionTokenIdx,
+        string memory redemptionName,
+        string memory redemptionSymbol,
```

Comment:
Can we please just hard code these. There's no reason imo to provide them as args given the whole system is literally defined for this exact token